artifactory/commands/transferfiles/manager.go

```diff
@@ -188,30 +189,41 @@ func (ptm *PollingTasksManager) stop() {
 }
 
 type producerConsumerDetails struct {
```

Comment:
producerConsumerDetails --> producerConsumerWrapper
Let's also change the names of the instances we create for this struct throughout the code, to reglect the new type name - ...Details --> ...Wrapper